libsolidity/formal/SMTChecker.cpp

```diff
@@ -472,20 +489,19 @@ void SMTChecker::checkCondition(
 	}
 	smt::CheckResult result;
```

Comment:
Can you add an assertion that `_descResult` is either satisfiable or unsatisfiable?